packages/init/src/steps/display-final-message.ts

```diff
@@ -2,12 +2,30 @@ import { CONSTS } from "@slicemachine/core";
 import * as NodeUtils from "@slicemachine/core/build/node-utils";
 import { logs } from "../utils";
 
-export function displayFinalMessage(cwd: string): void {
+export function displayFinalMessage(
+  cwd: string,
+  wasStarter: boolean,
+  reponame: string,
+  base: string
+): void {
   const yarnLock = NodeUtils.Files.exists(NodeUtils.YarnLockPath(cwd));
   const command = `${yarnLock ? "yarn" : "npm"} run ${CONSTS.SCRIPT_NAME}`;
-
   console.log();
```

Comment:
long forgotten console log ? haha

Reply:
It's there on purpose right?
I though it was making a new line :/

Reply:
haha no idea, if you think it's useful leave it, doesn't hurt anyone 😂